getSQL/settings.py

```diff
@@ -88,7 +88,7 @@
         'HOST': '127.0.0.1',
         'PORT': '3306',
         'USER': 'root',
-        'PASSWORD': '',
+        'PASSWORD': 'Healthy@123',
```

Comment:
Can you reset the password? Else one of us have to keep on updating this file everytime we need to run the app. 

I cant do it coz i have other dependencies on mysql.

Reply:
ok changed